src/manifests/distribution.py

```diff
@@ -0,0 +1,26 @@
+import logging
```

Comment:
To be consistent I would make this a class called `Distribution` that takes a base url and product name in the constructor, then add a `find` method.

Needs a test.

Reply:
Lets fast follow on these so the build isn't blocked, https://github.com/opensearch-project/opensearch-build/issues/1289